x-pack/plugins/secops/public/components/timeline/body/sort/sort_indicator.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as React from 'react';
+import { pure } from 'recompose';
+import { SortDirection } from './sort';
+
+export const SORT_DIRECTION_ASCENDING = '^';
+export const SORT_DIRECTION_DESCENDING = 'v';
+export const SORT_DIRECTION_NONE = '';
+
```

Comment:
Wouldn't these symbols be part of an enum like you have with the SortDirection. Something like:

```ts
/** Specifies a column's sort direction */
export type SortDirectionIndicator = '' | '^' | 'v';
```